src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/AuditService.kt

```diff
@@ -271,6 +277,7 @@ class AuditService(
     changes: Map<String, Any>,
   ): AuditEvent {
     val authUsername = SecurityContextHolder.getContext().authentication.name
+    println(authUsername)
```

Comment:
do we want to print the username in logs?

Reply:
removed this